lib/modules/dosomething/dosomething_northstar/dosomething_northstar.auth.inc

```diff
@@ -42,5 +42,12 @@ function dosomething_northstar_openid_connect_claims_alter(&$claims) {
  * @param string $provider - The provider used to authenticate
  */
 function dosomething_northstar_openid_connect_post_authorize($tokens, $account, $userinfo, $provider) {
-  // @TODO: Persist these in the database.
+  $edit = [];
+  dosomething_user_set_fields($edit, [
```

Comment:
do we need an if (module_exists) around this stuff? not sure if the fields always exist or if they're only created if the module is enabled